GVRf/Framework/framework/src/main/java/org/gearvrf/GVRActivity.java

```diff
@@ -218,10 +219,13 @@ protected void onDestroy() {
             mDockEventReceiver.stop();
         }
 
+        handleOnUndock();
```

Comment:
Calling onUndock is not there yet. @sivapenke has raised an issue about it. Will continue the discussion in the other repo.